### PR TITLE
typing

### LINKED ISSUE #24: Add typings
As we support py3.5+ would be nice to add type hints

### PR DESCRIPTION
**Pull request by pavivin**
Closes #24

### CODE REVIEW

**Comment by pavivin** [comment]
@hzlmn

**Comment by hzlmn** [comment]
Hi, thanks for Pr. certainly typings is  something i would like to add. I will review it later this week.

File: aiohttp_jwt/permissions.py [hunk 1]
```diff
@@ -41,19 +42,19 @@ async def wrapped(*args, **kwargs):
...
 def check_permissions(
-    scopes,
-    permissions_property='scopes',
-    comparison=match_all,
-):
-    if not callable(comparison):
-        raise TypeError('comparison should be a func')
+    scopes: Union[str, List[str]],
+    permissions_property: str = 'scopes',
+    comparsion: Callable = match_all,
```
Comment (hzlmn) [inline aiohttp_jwt/permissions.py]: Here and everywhere else small typo `comparsion ` should be `comparison`

File: aiohttp_jwt/middleware.py [hunk 2]
```diff
@@ -9,22 +10,22 @@
 
 logger = logging.getLogger(__name__)
 
-_request_property = ...
+_request_property: str = ''
```
Comment (hzlmn) [inline aiohttp_jwt/middleware.py]: Here should be `Union[Ellipsis, str]` or further checks won't work

File: aiohttp_jwt/permissions.py (hunk 1 above)
Comment (hzlmn) [reply aiohttp_jwt/permissions.py]: Also would be nice to define input/output definition for `Callable`

**Review by hzlmn — CHANGES_REQUESTED** [review]

File: aiohttp_jwt/middleware.py (hunk 2 above)
Comment (pavivin) [reply aiohttp_jwt/middleware.py]: I added ellipsis but 

`payload = request.get(middleware._request_property)`

Argument 1 to "get" of "Mapping" has incompatible type "Union[ellipsis, str]"; expected "str"mypy(error)

**Comment by pavivin** [comment]
@hzlmn, check new commit

**Pull request closed without merging**